src/components/Autocomplete.vue

```diff
@@ -200,12 +203,19 @@ export default {
       this.selectedIndex = null
       switch (typeof this.source) {
         case 'string':
+        case 'function':
           // No resource search with no input
           if (!this.display || this.display.length < 1) {
             return
           }
           this.loading = true
-          this.resourceSearch()
+
+          if (typeof this.source === 'string') {
```

Comment:
I think this would read better within their own switch case instead of sharing the `string` and `function` block

Reply:
Sure; pushed.

FWIW, there didn't seem to be an obviously best approach. I considered it, and combined the cases because I wanted to avoid duplicating the `this.display` short-circuit, and I also wanted to hint to the reader that `string` and `function` sources are the same "conceptually".  But I agree, it's ugly to re-test the switch expression.